src/lib/env.js

```diff
@@ -189,8 +188,8 @@ module.exports = function envFactory() {
   }
 
   function getOrDieFactory(f) {
-    return function(fullKeyName, $typeConverter /* args */ ) {
-      var value = getOrElse(fullKeyName, null, $typeConverter);
+    return function(fullKeyName, $typeConverter, _, $secure /* args */ ) {
```

Comment:
why pass `_` here? It's unused (old code?)

Reply:
I think it's because you called it with the `aliases` parameter in third position here: https://github.com/FGRibreau/common-env/blob/2bb1d361419615b15c7565c2915f1a2762518ad4/src/lib/env.js#L27
And you don't use it in the implementation but I needed the fourth parameter...

Reply:
hum, let's call it aliases then :)

Reply:
It's done!